testsuite/integration/basic/pom.xml

```diff
@@ -3418,6 +3418,44 @@
             </build>
         </profile>
 
+        <!-- Security manager. -->
+        <profile>
+            <id>ts.security.manager</id>
+            <activation>
+                <property>
+                    <name>security.manager</name>
+                </property>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>basic-integration-default-full.surefire</id>
+                                <configuration>
+                                    <excludes>
+                                        <!-- Hibernate Search is not compatible with the security manager. Never has been. -->
+                                        <exclude>org/jboss/as/test/integration/hibernate/search/**</exclude>
+                                    </excludes>
+                                </configuration>
+                            </execution>
+                            <execution>
+                                <id>basic-integration-default-web.surefire</id>
+                                <configuration>
+                                    <excludes>
```

Comment:
I believe adding 'combine.children="append"' will fix the CI failure. This profile is superseding the entire list of excludes from this execution instead of just adding to it.

Reply:
Never mind; that doesn't work.

There are only 3 tests involved here so it's better to drop the pom stuff and add @BeforeClass methods that call org.jboss.as.test.shared.util.AssumeTestGroupUtil.assumeSecurityManagerDisabled()

Reply:
> There are only 3 tests involved here so it's better to drop the pom stuff and add @BeforeClass methods that call org.jboss.as.test.shared.util.AssumeTestGroupUtil.assumeSecurityManagerDisabled()

Done; thanks for the hint, I was looking for something like this :)